package.json

```diff
@@ -35,6 +35,7 @@
     "client"
   ],
   "devDependencies": {
+    "@types/node-fetch": "^2.6.11",
```

Comment:
Let's make this a production dependency so that consuming packages can access this types package. I don't think that's possible with dev dependencies?